training-modules.Rproj

```diff
@@ -13,3 +13,6 @@ RnwWeave: Sweave
 LaTeX: pdfLaTeX
 
 AutoAppendNewline: Yes
+StripTrailingWhitespace: Yes
+
+MarkdownWrap: Sentence
```

Comment:
this is cute, I don't mind it!

Reply:
This was actually me almost adding in the "Write canonical visual mode markdown in source mode" option, but I don't like the way it renders bulleted lists, so I backed off (It adds 4 spaces after a dash). Note that this is only for the training-modules project that we (or at least I) use for editing material. This file is removed before we give it to users to avoid "project within project" confusion.

Also, the `MarkdownWrap` only applies to visual mode, so it is really kind of moot for us.

Reply:
> It adds 4 spaces after a dash

👎